src/staging/bot.py

```diff
@@ -102,9 +126,421 @@ async def _fetch_bci_devel_project_config(
             return await response.text()
 
 
-class _ProjectConfigs(TypedDict):
-    meta: ET.Element
-    prjconf: str
+@unique
+class ProjectConfig(Enum):
+    PRJCONF = "prjconf"
+    META = "prj"
+
+
+_PACKAGE_GROUP_NAME = "slcc_packages"
+
+
+@dataclass(frozen=True)
+class TripleZeroPackageGroups:
```

Comment:
can we split all those TripleZero packages groups commits to the bot into a separate PR? it's messy enough as is to review this and it seems entirely orthogonal to SLCC enablement.